src/app/notes/[noteId]/page.tsx

```diff
@@ -1,23 +1,13 @@
-import { getOneNote } from '@/app/notes/notesClient/notesApi';
+import { getOneNote } from '@/app/notes/notesActions';
+import { NoteBox } from '@/app/notes/[noteId]/NoteBox';
 
 const Note = async ({ params }: { params: { noteId: string } }) => {
   const note = await getOneNote(params.noteId);
 
   return (
-    <form>
-      <input
-        id="title"
-        type="text"
-        className="m-3 text-[22px] bg-transparent w-full"
-        defaultValue={note.title}
-      />
-      <input
-        id="text"
-        type="text"
-        className="m-3 bg-transparent w-full"
-        defaultValue={note.text}
-      />
-    </form>
+    <div>
```

Comment:
nit: why do we surround `<form>` with a `<div>`? And if we need to, what do you think of making it a part of `NoteBox`?

Reply:
for no reason 🤡